-meeting-template.md

```diff
@@ -1,57 +1,54 @@
 # YYYY-MM-DD Hypha Worker Co-op: All Hands Meeting
 
-<sup>[from template][template]</sup>
+<sup>[from template][template] | [:notebook: Meetings][meetings] | [:date: Calendar][calendar] | [:white_check_mark: Tasks][tasks] | [:cat: Code][gh] | [:open_file_folder: Drive][gdrive]</sup>
 
-[:date: Calendar][calendar] | [:cat: GitHub][gh] | [:open_file_folder: GDrive][gdrive] | [:notebook: Meeting Index][meetings]
-
-Time: ...  
-Location: https://meet.jit.si/hyphacoop  
-Attending: ...
-
-Notetaker: ...
-
-:raising_hand: Next: ... COPY FROM PREVIOUS WEEK  
-<sup>Bump to end on taking turn :)</sup>
```

Comment:
why remote this?

Reply:
We are all familiar with the turn by now and want to condense the lines before actual notes. The many lines at the top feel disorienting.